sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -835,6 +971,9 @@ Le gestionnaire de contrôle des services a tenté de prendre une action correct
 Language=Polish
 Menedżer sterowania usługami próbował podjąć akcję korekcyjną (%2) po nieoczekiwanym zakończeniu usługi %3, ale ta akcja nie powiodła się przy następującym błędzie: %n%4.
 .
+Language=Romanian
+Administratorul de control al serviciului a încercat să ia o acțiune corectivă (%2) după o închidere neanunțată a serviciului %3, dar această acțiune a eșuat cu următoarea eroare: %n%4
```

Comment:
In the previous translated resource, "unexpected" got translated as "neașteptat" whereas here is translated as "neanunțată" (which is less accurate). Please revise.